packages/e2e/test/artillery/StakePoolSearch.yml

```diff
@@ -6,12 +6,11 @@ config:
     timeout: 180
   phases:
     - name: 'Stake Pool Search'
-      duration: 10
-      arrivalRate: 1
+      duration: 1
+      arrivalCount: 1
```

Comment:
Is this 1 VU enough for the phase?

Reply:
Yes, that's the phase for the first user, the first user needs to perform a query without filters to fetch all the pools